src/tests/ftest/container/create.py

```diff
@@ -56,19 +56,14 @@ def test_container_create(self):
         if handleparam == 'VALID':
             poh = self.pool.pool.handle
         else:
-            self.cancel("skipping this test case until DAOS-4099 is fixed")
```

Comment:
this part of the patch is from another PR that has not yet landed, PR 1933. That one will land first and a merge with master will be done.